### DISCUSSION TITLE
Traefik v2.7 + Cockpit 273 seems to cause WSS problems

### DISCUSSION THREAD

**Discussion by divStar** [cockpit-project/cockpit#17681] [General]
**Assuming your set up worked e.g. in June, 2022, with Traefik v2 and Cockpit,** this set up may just stop working, erroring out after you logged in into Cockpit using your internal domain (e.g. `cockpit.my.domain`). I have not changed anything about my setup of Cockpit or Traefik, I have been using TLS / SSL (https) for the subdomain (e.g. `https://cockpit.my.domain`) successfully for quite a few months.

Yesterday I noticed, that I couldn't successfully log in into Cockpit anymore. When removing `Origins` from `cockpit.conf`, I could login directly using `http://<IP>:<PORT>` just fine. The error seemed to be related to the web console not being able to successfully perform the `wss` connection. I got a status code 403 (Forbidden). After making sure SSL certificates were up-to-date, I got a 302 (Found) response, but login still would not work (I received a page stating there was an unexpected interal error and thus the connection couldn't be established).

As a last resort I decided to move from Traefik v2.7 to the latest (which currently seems to be v2.8.3). Suddenly Cockpit started working as it used to.

**My take away here is: IF you happen to suddenly have problems with Traefik v2 + Cockpit, make sure Traefik v2 and Cockpit are both up-to-date (currently Trafik v2.8.3 + Cockpit 273 work just fine) or else you might run into weird issues.**

**Comment by fsdrw08** [comment]
Hi, could you share your traefik and cockpit config here?
